modin/tests/pandas/dataframe/test_join_sort.py

```diff
@@ -140,7 +141,11 @@ def test_join(test_data, test_data2):
                 lsuffix="_caller",
                 rsuffix="_other",
             )
-            df_equals(modin_result, pandas_result)
+            if sorts[j]:
+                # sorting in `join` is implemented through range partitioning technique
```

Comment:
I don't quite understand this comment. sorting is just called in df_equals_and_sort. Should we remove this comment?

Reply:
I expanded the comment a little